pandas/tests/indexes/multi/test_monotonic.py

```diff
@@ -182,22 +183,28 @@ def test_searchsorted_monotonic(indices):
     # test searchsorted only for increasing
     if indices.is_monotonic_increasing:
```

Comment:
pls don't import is_scalar from testing, import it from the canonical location

pandas.api.types.is_scalar.